client.go

```diff
@@ -105,6 +133,59 @@ func handleHTTPError(httpResponseData []byte, httpResponse *http.Response, authe
 	}
 }
 
+// Populate OrganizationID with the first organization found in the account
```

Comment:
I just don't think we should be populating the first organization automatically. It should be configured purposefully in the client config.